src/main/java/openfasttrack/importer/tag/TagImporter.java

```diff
@@ -82,15 +86,21 @@ public void runImport()
     private void processLine(final int lineNumber, final String line)
     {
         final Matcher matcher = this.tagPattern.matcher(line);
+        int counter = 0;
         while (matcher.find())
         {
             this.listener.beginSpecificationItem();
-            final SpecificationItemId id = SpecificationItemId.parseId(matcher.group(1));
+            final String generatedName = this.fileName + ":" + lineNumber + "-" + counter;
+            final SpecificationItemId generatedId = SpecificationItemId.createId(matcher.group(1),
+                    generatedName, 0);
```

Comment:
":" and "-" are not allowed in IDs. I admit that we probably should allow "-". Used in URIs ":" is a character with a special meaning though.

Also file names can contain non-ASCII characters and white spaces which we would need to mangle if we want to create an ID from it.

Instead deduct the ID from the covered ID instead since we keep file and line in meta fields anyway:

Input: `[impl->dsn~tag_import.auto_id~1]`
Generated ID: `impl~tag_import.auto_id~1` covers `dsn~tag_import.auto_id~1`

This way you can tell from the ID what the implementation is supposed to do.